### ISSUE TITLE
Accessibility: Implement the Warning Colour Preference as Standard in MegaMek

### ISSUE THREAD

**Issue by Windchild292** [MegaMek/megamek#1897]
The current warning colour is red, which cannot be easily seen for people with certain forms of colourblindness. #1859 implemented a value in GUIPreferences for the Warning Colour, but that is only implemented in the board editor. This story is to implement it across MegaMek.

**Comment by gsparks3** [comment]
I think this issue was missed when the PR went in, but a variety of text highlight colors are now adjustable in Megamek Client Settings. 

![Image](https://github.com/user-attachments/assets/221b8457-3ed0-4ca1-84a2-5d6f6c330550)
